deps/spidershim/test/value.cc

```diff
@@ -778,6 +778,24 @@ TEST(SpiderShim, Date) {
   EXPECT_EQ(time, Date::Cast(*date.ToLocalChecked()->ToObject())->ValueOf());
 }
 
+namespace {
```

Comment:
> I looked at those, but it wasn't clear at first glance how to port them. I'll take another look, however.

To close the loop on this: I took another look, and it's more clear how to port those tests now, but it isn't clear how useful it is to do so, given that they mostly test features that we haven't implemented yet (and that Node doesn't use). Nevertheless, I've started porting them, with the first two in #59.